src/posts/2020-04-16-continuous-stuff-with-github-actions/index.md

```diff
@@ -0,0 +1,262 @@
+---
+id: 147
+path: /posts/continuous-stuff-with-github-actions
+title: "Continuous Stuff with Github Actions"
+author: miguel-palhas
+date: 2020-04-20
+tags:
+  - devops
+intro: "Last year, I took on the task of improving the continuous process over at Utrust. We weren't really happy with the amount of work that went into our releases, and I was looking for a more agile approach, where everyone from developers to the QA team could do their part with low friction."
```

Comment:
You can turn this into a multiline string to prevent the wrapping in your editor and make it easier to edit. (https://yaml-multiline.info/)

```suggestion
intro: >
  Last year, I took on the task of improving the continuous process over at
  Utrust. We weren't really happy with the amount of work that went into our
  releases, and I was looking for a more agile approach, where everyone from
  developers to the QA team could do their part with low friction.
```